MCP-Cursor-راهنمای-فارسی.md

```
@@ -0,0 +1,2915 @@
+3. پاسخ به سوالات در انجمن‌ها
+
+## منابع مفید
+
+### مستندات رسمی
+- [Model Context Protocol Specification](https://modelcontextprotocol.io/)
+- [Cursor Documentation](https://cursor.sh/docs)
+- [FastMCP Documentation](https://github.com/fastmcp/fastmcp)
+- [MCP SDK Documentation](https://github.com/anthropics/model-context-protocol)
+
+### آموزش‌های ویدیویی
+- [دوره جامع MCP در Cursor (فارسی)](https://example.com/mcp-course-fa)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)